commercetools-models/src/test/java/io/sphere/sdk/cartdiscounts/CartDiscountFixtures.java

```diff
@@ -57,6 +58,13 @@ public static void withCartDiscount(final BlockingSphereClient client, final Una
         client.executeBlocking(CartDiscountDeleteCommand.of(cartDiscount));
     }
 
+    public static void withCartDiscount(final BlockingSphereClient client, final UnaryOperator<CartDiscountDraftBuilder> builderUnaryOperator, final Function<CartDiscount, CartDiscount> update) {
```

Comment:
@rjsvaljean Just a small enhancement, actually we can use `UnaryOperator<CardDiscount>` instead of `Function<CartDiscount, CartDiscount>`